content/en/docs/installation/kubernetes.md

```diff
@@ -92,9 +92,9 @@ kubectl create clusterrolebinding cluster-admin-binding \
 As an alternative to the YAML manifests referenced above, we also provide an
 official Helm chart for installing cert-manager.
 
-### Pre-requisites
+### Prerequisites
 
-- Helm_ and Tiller installed (or alternatively, use `Tillerless Helm v2`_)
+- Helm and Tiller installed (or alternatively, use `Tillerless Helm v2`z)
```

Comment:
Underscores were there to make these words links - also the 'z' at the end seems like a typo